utilities/tag_repo_visibility/tag_for_visibility_GH.py

```diff
@@ -0,0 +1,76 @@
+import requests
+import json
+
+# GitHub personal access token
+GITHUB_TOKEN = 'your_github_token'
+# GitHub organization
+GITHUB_ORG = 'your_org'
+# GitHub API base URL
+GITHUB_API_URL = 'https://api.github.com'
+#Semgrep Organization Slug (Found by going to the Semgrep settings and scrolling to the bottom).
+SEMGREP_ORG_SLUG = 'your_semgrep_org_slug'
+#Semgrep API token.
+SEMGREP_API_TOKEN = 'your_semgrep_api_token'
```

Comment:
Maybe let's pull the secrets from environment variables with `os.getenv("GITHUB_TOKEN")` and the like? Not a huge deal for something that's only meant to be run once, but to maintain the tags for new repos it could get automated, and it'd feel pretty silly for semgrep to detect secrets in code they got from semgrep. :)

Reply:
(If you do this don't forget to update the instructions, which is why I didn't just make a code suggestion.)

Reply:
Usually I leave this kind of thing for the customer to operationalize. 

If they want to run this in an automated fashion, usually they'll have a secrets store or have these variables available within the pipeline. os.getenv would be inappropriate for these scenarios so they'll have to make some customization anyways. 

I can change this to use the dotenv module if you think it would be appropriate to store these values in a .env file for appearances!